config/environments/development.rb

```diff
@@ -58,6 +58,8 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
+  config.hosts << 'b6a5-82-0-200-55.ngrok-free.app'
```

Comment:
Does this need to stay long term?

Reply:
ah no, sorry, probably me doing an inadvertant commit all 
This was when I was trying to do screenreader testing through a VM, which couldn't access my local network, so was using ngrok as a proxy.
I'll remove it